addons/html_editor/static/src/main/powerbox/search_powerbox_plugin.js

```diff
@@ -82,11 +87,10 @@ export class SearchPowerboxPlugin extends Plugin {
     }
     openPowerbox() {
         const selection = this.shared.getEditableSelection();
-        this.offset = selection.startOffset;
-        const historySavePointRestore = this.shared.makeSavePoint();
+        this.offset = selection.startOffset - 1;
         this.shared.openPowerbox({
             commandGroups: this.commandGroups,
-            onApplyCommand: historySavePointRestore,
+            onApplyCommand: () => this.historySavePointRestore(),
```

Comment:
why the fat arrow? the method `this.historySavePointRestore` should be properly binded already